docs/src/HowToGuides/Atmos/Microphysics.md

```diff
@@ -1,125 +1,279 @@
-|``\tau_{acnv}``       | cloud to rain water autoconversion timescale              | ``s``                    | ``10^3``              |
-|``q_{liq\_threshold}``| cloud to rain water autoconversion threshold              | -                        | ``5 \cdot 10^{-4}``   |
-|``E_{col}``           | collision efficiency between rain drops and cloud droplets| -                        | ``0.8``               |
-|``C_{drag}``          | rain drop drag coefficient                                | -                        | ``0.55``              |
-|``a_{vent}, b_{vent}``| rain drop ventilation factor coefficients                 | -                        | ``1.5 \;``,``\; 0.53``|
-|``K_{therm}``         | thermal conductivity of air                               | ``\frac{J}{m \; s \; K}``| ``2.4 \cdot 10^{-2}`` |
-|``\nu_{air}``         | kinematic viscosity of air                                | ``\frac{m^2}{s}``        | ``1.6 \cdot 10^{-5}`` |
-|``D_{vapor}``         | diffusivity of water vapor                                | ``\frac{m^2}{s}``        | ``2.26 \cdot 10^{-5}``|
-
-## Rain drop size distribution
-
-The rain-drop size distribution is assumed to follow Marshall-Palmer distribution (Marshall Palmer 1948 eq. 1):
 ```math
-\begin{equation}
-n(r) = n_{0_{MP}} exp\left(- \lambda_{MP} \, r \right)
-\end{equation}
+m(r) = \alpha r^{\beta}
+```
+```math
+a(r) = \gamma r^{\delta}
+```
+```math
+v_{term}(r) = \zeta r^{\eta}
 ```
 where:
- - ``r`` is the drop radius,
- - ``n_{0_{MP}}`` and ``\lambda_{MP}`` are the Marshall-Palmer distribution parameters (twice the values used in the Marshall Palmer 1948, because we use drop radius and not diameter).
+ - ``r`` is the particle radius,
+ - ``\alpha, \, \beta, \, \gamma, \, \delta, \, \zeta, \, \eta \,`` are the coefficients.
+
+These coefficients,
+smiliarliy to all other microphysics parameters,
+are not hardcoded in the final microphysics parameterizations.
+The goal is to allow easy flexibility when calibrating the microphysics parameters.
+With that said, the assumption about the shape of the particles is used twice
+when deriving the microphysics formulae:
+ - The mass transfer equation (\ref{eq:mass_rate}) used in snow autoconversion, rain evaporation,
+   snow sublimation and snow melt rates is derived assuming spherical particle shapes.
+   To correct for non-spherical shape it should be multiplied by a function of the
+   particle aspect ratio.
+ - The geometric collision kernel used for deriving rain-snow accretion rate
+   assumes that both colliding particles are spherical.
+   It does not take into account the reduced cross-section of snow particles that is used
+   when modeling snow - cloud water and snow - cloud ice accretion.
+
+|    symbol          |         definition                              | units                    | default value                         |
```

Comment:
I think adding another column to all of these tables with the references would be cleaner.

Reply:
Sounds good. I'll do that.